### PR TITLE
Adding instructions for using a Group Managed Service Account.

### PR DESCRIPTION
**Pull request by phillipsj**
Currently, the documentation doesn't document how to use a gMSA to to connect to SQL Server. This attempts to add some basic guidance on how to configure it to work.

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/microsoft-sql/microsoft-sql-server-integration.mdx [hunk 1]
```diff
@@ -54,7 +54,7 @@ For a comprehensive list of specific Windows versions, check the table of [compa
 * The [infrastructure agent](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent-new-relic) installed.
 * Install the [infrastructure agent](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent-new-relic) on a Linux or Windows OS host, or on a host capable of remotely accessing where SQL Server is installed.
 
-* Requires SQL Server user or domain user with [user privileges](#users-privileges) for both `CONNECT` and `VIEW SERVER STATE`, and `READ` access permissions.
+* Requires SQL Server user, domain user, or Group Managed Service Account with [user privileges](#users-privileges) for both `CONNECT` and `VIEW SERVER STATE`, and `READ` access permissions.
```
Comment (akristen) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/microsoft-sql/microsoft-sql-server-integration.mdx:57]:
```suggestion
* Requires SQL Server user, domain user, or group managed service account with [user privileges](#users-privileges) for both `CONNECT` and `VIEW SERVER STATE`, and `READ` access permissions.
```

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/microsoft-sql/microsoft-sql-server-integration.mdx [hunk 2]
```diff
@@ -93,6 +93,10 @@ To install the Microsoft SQL Server integration, follow the instructions for you
 3. Go to `C:\Program Files\New Relic\newrelic-infra\integrations.d\` and rename the `mssql-config.yml.sample` file to `mssql-config.yml`.
 4. Edit the `mssql-config.yml` configuration file with your favorite editor. Check out some [configuration file examples.](#examples).
 
+<Callout variant="important">
+  If using a Group Managed Service Account, gMSA, for SQL Server access, the agent needs to run as the gMSA.
```
Comment (akristen) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/microsoft-sql/microsoft-sql-server-integration.mdx:97]:
```suggestion
  The agent needs to run as the gMSA if it uses a group managed service account for SQL Server access.
```

Comment (akristen) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/microsoft-sql/microsoft-sql-server-integration.mdx:97]: @phillipsj can you explain what this call out means, actually? I'm understanding that the agent itself needs to be configured to access a group managed service account (therefore letting it run as a gMSA) if the SQL server itself is managed by a group-type account on the mySQL-side? Is that correct?

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/microsoft-sql/microsoft-sql-server-integration.mdx [hunk 3]
```diff
@@ -200,6 +204,44 @@ Create a new login and grant the necessary permissions:
...
 
   </Collapser>
+
+  <Collapser
+    id="domain-user"
+    title="For Windows Group Managed Service Account"
+  >
+
+
+    1. Create a new login and grant `CONNECT` and `VIEW` permissions:
```
Comment (akristen) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/microsoft-sql/microsoft-sql-server-integration.mdx:214]: Are they creating a new login on the SQL side, or on the new relic side? Where are they adding this snippet to grant connect and view permissions? @phillipsj

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/microsoft-sql/microsoft-sql-server-integration.mdx [hunk 4]
```diff
@@ -200,6 +204,44 @@ Create a new login and grant the necessary permissions:
...
+
+    ```sql
+    USE master;
+      CREATE LOGIN [DomainName\gMSA] FROM windows;
+      GRANT CONNECT SQL TO [Domain\gMSA];
+      GRANT VIEW SERVER STATE TO [Domain\gMSA];
+      GRANT VIEW ANY DEFINITION TO [Domain\gMSA];
+    ```
+
+    2. Use the following statements to grant `READ` access privileges to the user:
```
Comment (akristen) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/microsoft-sql/microsoft-sql-server-integration.mdx:224]: Where are they using the following statements? Is it immediately after the snippet that grants CONNECT and VIEW permissions? Is it later in some kind of file? What file are these updates located in?

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/microsoft-sql/microsoft-sql-server-integration.mdx [hunk 5]
```diff
@@ -261,6 +303,28 @@ The configuration file has common settings applicable to all integrations like `
 
   </Collapser>
 
+  <Collapser
+    id="basic-config-gmsa-domain"
+    title="Basic configuration with Windows Group Managed Service Account"
+  >
+    This is the basic configuration used to collect metrics and inventory from your localhost. In this case we use a Group Managed Service Acccount for authentication which requires a blank username and password:
```
Comment (akristen) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/microsoft-sql/microsoft-sql-server-integration.mdx:310]: I see we're using gMSA interchangeable with Group Managed Service Account. Is gMSA just the acronym for this account type?

**Comment by akristen** [comment]
Thanks so much for your contribution @phillipsj! I left some follow up comments to clarify the instructions, otherwise this is amazing stuff!

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/microsoft-sql/microsoft-sql-server-integration.mdx (hunk 2 above)
Comment (phillipsj) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/microsoft-sql/microsoft-sql-server-integration.mdx:97]: @akristen I can see that being confusing. If you require services to access a SQL Server using a gMSA, then the service, like the New Relic Agent, needs to be running as that account.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/microsoft-sql/microsoft-sql-server-integration.mdx (hunk 3 above)
Comment (phillipsj) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/microsoft-sql/microsoft-sql-server-integration.mdx:214]: This is for creating the login on the SQL Server side, not in New Relic.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/microsoft-sql/microsoft-sql-server-integration.mdx (hunk 4 above)
Comment (phillipsj) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/microsoft-sql/microsoft-sql-server-integration.mdx:224]: This is just the setup of the gMSA on the SQL Server side. I was just following the pattern for creating the domain accounts or the SQL login account.

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/microsoft-sql/microsoft-sql-server-integration.mdx (hunk 5 above)
Comment (phillipsj) [reply src/content/docs/infrastructure/host-integrations/host-integrations-list/microsoft-sql/microsoft-sql-server-integration.mdx:310]: gMSA is just the acronym for a Group Managed Service account. This is a Microsoft concept that can be used in Windows and SQL Server.

**Comment by phillipsj** [comment]
@akristen Insee you closed this without it being merged.  I did the updates requested, was there something else I needed to do?

**Comment by akristen** [comment]
@phillipsj I started a new PR with these changes so that the branches were up to date without any major merge conflicts. Thought I'd be able to c/p the text even with the PR closed, but I jumped the gun 😅  I'll reply with the new PR here once I have your suggestions staged 🫡

**Comment by akristen** [comment]
Since this PR is quite old, I've remade it over in this PR so it's more in sync with our most recent develop branch: https://github.com/newrelic/docs-website/pull/18745

**Pull request closed without merging**